src/net/client/auth/JwtBearerSession.cpp

```
@@ -0,0 +1,212 @@
+{
+  if (refresh_token.empty())
+    return false;
+
+  LogFmt("{}: refreshing access token", config.log_tag);
+
+  std::string cookie = config.refresh_cookie_name;
+  cookie += '=';
+  cookie += refresh_token;
+
+  CurlSlist headers;
+  headers.Append("Content-Type: application/json");
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function declared not to throw exceptions.

(throwInNoexceptFunction)